test/tests/ncp-lms/tests

```diff
@@ -8,26 +10,6 @@
     expect_out = '0 of 22 singular values'
     custom_cmp = nodal.cmp
   []
-  [interpolated_lm_ncp]
```

Comment:
Curious - why did you remove these tests (and the associated NCP objects)? Just another way of doing LM that you prefer?

Reply:
Since writing these tests a couple years ago (!!), I've gotten a lot more experience with NCP-type constraints in mechanical contact and the literature suggests performing the NCP constraints on essentially a per-dof basis. So if the LM is Lagrange, then the NCP constraint should be enforced on a per-node basis (which I do with `NodalKernels`). 

Just within the last week @jbadger95 has seen oscillatory behavior when trying to set an LM to zero in an "elemental" way as opposed to on a per-dof basis. You can imagine for example that if you do an integrated/elemental approach, an allowed solution to, say, \int_{elem} \lambda dV using a two point quadrature rule would be for the solution at qp1 to be +C and the solution at qp2 to be -C. We want to avoid these oscillatory solutions

Reply:
I understand! Thanks for the details :)